docs/application/native/guides/ui/efl/mobile/component-tooltip.md

```diff
@@ -6,7 +6,7 @@ This feature is supported in mobile applications only.
 
 ## Basic Usage
 
-> **Note**  
+> **Note**
```

Comment:
line 9: Note form was not applied.
line 75: Note form was not applied.

Reply:
done.